src/components/Card.js

```diff
@@ -18,7 +23,7 @@ const Card = ({
         <div className="d-flex align-center mt-8">
           <button type="button" className="btn">Comment</button>
           <div className="hr" />
-          <button type="button" className="btn" onClick={(() => { dispatch(removeBook(id)); })}>Remove</button>
+          <button type="button" className="btn" onClick={(() => { handleRemove(id); })}>Remove</button>
```

Comment:
Your remove function does not currently work as expected. The books are removed from the UI but are not removed from the API because when the page is refreshed, they come back. Please take a look at this and ensure it works properly🙏